test/TestSuite.hx

```diff
@@ -1,20 +1,20 @@
 import massive.munit.TestSuite;
 
-import massive.munit.AssertionExceptionTest;
 import massive.munit.AssertTest;
-import massive.munit.async.AsyncDelegateTest;
+import massive.munit.TestSuiteTest;
```

Comment:
sort please

Reply:
I didn't manually changed it it's after I run the build.sh